holidayextras/brands
# Tweaks/fixes to globals close, colours, utilities; hx buttons, panels.

#### What does this PR do? (please provide any background)
##### Globals brand
* Center-aligned `.close` button text.
* Added `@youtube-colour` and `@google-plus-colour` variables.
* Fixed `.fade-in-visibility()` and `.fade-out-visibility()` mixins. 
##### HX brand
* Fixed text hiding on social buttons and used `.button-variant()` mixin instead.
* Fixed margin resets for `h1` and `h2` within `.panel`.

#### How can this be tested?
`cd pattern-library && npm link ../brands`
Checkout the corresponding branch in Pattern Library to see the effects on the new hotel info page.

#### Any tech debt?
Headings margin resets need to be addressed consistently at a more global level.

#### Screenshots / Screencast
N/A - very subtle changes.

#### What gif best describes how you feel about this work?
![]()


---

**Reviewer**

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked all the work locally on pattern library.

---
